MIGRATION.md

```diff
@@ -1503,6 +1504,10 @@ Additionally to changing the docs information architecture, we've updated the AP
 
 The primary change of the `Meta` block is the ability to attach to CSF files with `<Meta of={}>` as described above.
 
+##### Primary block
+
+The `Primary` block now also accepts an `of` prop as described above. It still accepts being passed `name` or no props at all.
+
```

Comment:
These migration guides are between specific versions, adding this note here communicates that this was introduced in 7.0, while it's probably being introduced in 7.6.

I suggest you create a new top section below the Table Of Content called "From version 7.5.0 to 7.6.0", that includes this content. Given that this will move it "out" of the doc block section, you probably need to add a bit more context and link to this doc block section.

Reply:
Ok sounds good, I'll push a commit along with the other one updating the docs.